src/PrestaShopBundle/Controller/Admin/Configure/ShopParameters/OrderStateController.php

```diff
@@ -230,6 +262,14 @@ public function createOrderReturnStateAction(Request $request)
      */
     public function editOrderReturnStateAction(int $orderReturnStateId, Request $request)
     {
+        if ($this->get('prestashop.adapter.multistore_feature')->isActive()) {
+            $this->addFlash(
+                'info',
+                $this->trans('Note that this feature is available in all shops context only. It will be added to all your stores.',
+                    'Admin.Notifications.Info')
+            );
+        }
+
```

Comment:
Can you create a method to do the job, you repeat the same condition everytime

Reply:
I was not sure about that (I was afraid of doing premature abstraction) but if you believe it's better, then let's do it 👍

Reply:
Done in https://github.com/PrestaShop/PrestaShop/pull/20442/commits/4153ab72cfb6a663bf7c67452c710bb6fd4f6feb